Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+            "company_name": profile.company_name,
+            "website": profile.website,
+            "industry": profile.industry,
+            "contact_person": profile.contact_person,
+            "contact_email": profile.contact_email,
+            "created_at": t
+        }).execute()
+        
+        if response.data:
+            return BrandProfileResponse(**response.data[0])
+        else:
+            raise HTTPException(status_code=400, detail="Failed to create brand profile")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)